pressbooks/pressbooks-book
# fix: CTA dismiss button

Fixes pressbooks/pressbooks#2958

This PR aims to fix an issue when trying to add an event listener to an element that does not exist anymore. The issue happens when updating the network settings to not display the default CTA banner.
<img width="623" alt="image" src="https://user-images.githubusercontent.com/1761690/197059238-4d186002-d288-4a12-8426-ff89430c205b.png">

When visiting a book page, the banner is not displayed as expected but the page still tries to add an event listener to the CTA button causing the code to break
<img width="643" alt="image" src="https://user-images.githubusercontent.com/1761690/197059465-06784965-f09c-42fe-907a-bca03c754389.png">

**How to test**

1. Disable the CTA banner on the network settings
2. Visit a book page notice the error message
3. Switch to this branch and make sure the error message is not there when you visit the book again
4. Enable the CTA banner again and make sure the banner is dismissible
